troposphere/autoscalingplans.py

```diff
@@ -0,0 +1,82 @@
+from . import AWSObject, AWSProperty
```

Comment:
Thank you for redoing the PR. Please add a copyright header using the form in the other files (either substitute my name/email with yours or just use mine). Thanks!

Reply:
Added copyright.